src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -80,22 +79,12 @@ export class Table extends PureComponent<TableProps, TableState> {
     isExportPopoverOpen: false,
     isIncludeReferencesDeepChecked: true,
     activeAction: undefined,
-    isColumnDataLoaded: false,
   };
 
   constructor(props: TableProps) {
     super(props);
   }
 
-  componentDidMount() {
-    this.loadColumnData();
-  }
-
-  loadColumnData = async () => {
-    await Promise.all(this.props.columnRegistry.getAll().map((column) => column.loadData()));
-    this.setState({ isColumnDataLoaded: true });
-  };
```

Comment:
I'm guessing `spaces` was the only consumer of this API/feature, and the usage was removed in this PR, right?

Reply:
Yes that's correct. I figured if we don't need it anymore, we should remove it.